GCRCatalogs/dc2_coadd.py

```diff
@@ -34,91 +84,303 @@ def _subclass_init(self, **kwargs):
 
         if not os.path.isdir(self._base_dir):
             raise ValueError('`base_dir` {} is not a valid directory'.format(self._base_dir))
-        self._datasets, self._columns = self._generate_native_datasets_and_columns()
-        if not self._datasets:
-            raise RuntimeError('No catalogs were found in `base_dir` {}'.format(self._base_dir))
 
         self._dataset_cache = dict()
 
-        self._quantity_modifiers = {
+        self._quantity_modifiers = self._generate_modifiers()
+        self._datasets, self._columns = self._generate_datasets_and_columns()
+        if not self._datasets:
+            err_msg = 'No catalogs were found in `base_dir` {}'
+            raise RuntimeError(err_msg.format(self._base_dir))
+
+    @staticmethod
+    def _generate_modifiers():
+        """Creates a dictionary relating native and homogenized column names
+
+        Returns:
+            A dictionary of the form {<homogenized name>: <native name>, ...}
+        """
+        modifiers = {
+            'objectId': 'id',
+            'parentObjectId': 'parent',
             'ra': (np.rad2deg, 'coord_ra'),
             'dec': (np.rad2deg, 'coord_dec'),
+            'centroidX': 'slot_Centroid_x',
+            'centroidY': 'slot_Centroid_y',
+            'centroidX_err': 'slot_Centroid_xSigma',
+            'centroidY_err': 'slot_Centroid_ySigma',
+            'centroid_flag': 'slot_Centroid_flag',
+            'psNdata': 'base_PsfFlux_area',
+            'extendedness': 'base_ClassificationExtendedness_value',
+            'HSM_res': 'ext_shapeHSM_HsmShapeRegauss_resolution',
+            'HSM_ell': (np.hypot, 'ext_shapeHSM_HsmShapeRegauss_e1', 'ext_shapeHSM_HsmShapeRegauss_e2'),
         }
 
+        modifiers['pass_basic_flag_cut'] = (
```

Comment:
`good`

Reply:
"reasonable" is more the sense, but that's too long to type.
"pass_basic_flag_cut" seems descriptive, but it's not particularly, because you still have to go look up what those basic cuts were.  So I think "good" might capture that as well.

The current star-galaxy separation isn't very good -- a significant number of barely resolved galaxies are classified as stars -- so we might just stick with `good` for now.

Reply:
+1 for `good`